runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -215,6 +232,53 @@ public void onConfigChanged(ConfigChanged event)
 		clientThread.invokeLater(() -> checkRaidPresence(true));
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (!config.showLootValue() ||
+			event.getGroupId() != WidgetID.CHAMBERS_OF_XERIC_REWARD_GROUP_ID ||
+			chestOpened)
+		{
+			return;
+		}
+		chestOpened = true;
+		ItemContainer rewardItemContainer = client.getItemContainer(InventoryID.CHAMBERS_OF_XERIC_CHEST);
+		if (rewardItemContainer == null)
+		{
+			return;
+		}
+
+		final Collection<Item> rewardItems = Arrays.stream(rewardItemContainer.getItems())
+			.filter(item -> item.getId() > 0)
+			.map(item -> new Item(item.getId(), item.getQuantity()))
+			.collect(Collectors.toList());
+
+		int rewardValue = 0;
+
+		for (Item reweardItem : rewardItems)
+		{
+			if (UNTRADABLE_ITEMS.contains(reweardItem.getId()))
```

Comment:
Why is this here?

Reply:
Guess it is not needed, was thinking about it not checking item price if it is an untradable item but the item prive would be 0 anyways so it is useless, will take it out

Reply:
The item manager has some logic to apply prices to untradeables, but if you don't want it to there is a `ignoreUntradeableMap` flag.

Reply:
Removed the check